HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -408,6 +408,14 @@ def customiseFor17098(process):
        producer.copyTrajectories = cms.untracked.bool(False)
     return process
 
+# Configurable Pixel Cluster Shape in LowPtSeedComparitor
+def customiseFor17393(process):
+    for producer in producers_by_type(process,"PixelTripletHLTEDProducer"):
+         if hasattr(producer,'SeedComparitorPSet'):
+             if (producer.SeedComparitorPSet.ComponentName.value() == 'LowPtClusterShapeSeedComparitor'):
+                  producer.SeedComparitorPSet.clusterShapeHitFilter = cms.string('ClusterShapeHitFilter')
```

Comment:
Hi, sorry for the delay. Yes, you are completely right. I will address it asap. Thanks for spotting the issue.